README.md

```diff
@@ -72,6 +72,9 @@ We use DataCite to mint DOIs and in production you must to define the `DATACITE_
 
 RDSS uses the same [release and deployment process](https://github.com/pulibrary/rdss-handbook/blob/main/release_process.md) for all projects.
 
+## Mail on Development
```

Comment:
This information would need to be up in the dependencies (https://github.com/pulibrary/pdc_describe/pull/872/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15).  This change requires the user to have mailcatcher installed and running on their local machine